### PR TITLE
docs(guide): Update GraphCMS data sourcing guide

### PR DESCRIPTION
**Pull request by ynnoj**
## Description

Updates the [Sourcing from GraphCMS](https://www.gatsbyjs.com/docs/sourcing-from-graphcms/) guide to use the new [`gatsby-source-graphcms`](https://github.com/GraphCMS/gatsby-source-graphcms) plugin.

### Documentation

N/A

## Related Issues

N/A

### CODE REVIEW

**Review by ronakganatra — DISMISSED** [review]

File: docs/docs/sourcing-from-graphcms.md [hunk 1]
```diff
@@ -4,158 +4,246 @@ title: Sourcing from GraphCMS
...
+### Add the `gatsby-source-graphcms` plugin
 
-Additionally, you need the `gatsby-source-graphql` library. Because GraphCMS uses GraphQL natively, you will take advantage of Gatsby's ability to simply stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
+In order to fetch data from your GraphCMS project, you will need to install [`gatsby-source-graphcms`](https://github.com/GraphCMS/gatsby-source-graphcms).
 
-You can install this component with:
+You can install this package with:
 
 ```shell
+<<<<<<< HEAD
```
Comment (notrab) [inline docs/docs/sourcing-from-graphcms.md:39]: But we're a HEADless API..... 🥁

File: docs/docs/sourcing-from-graphcms.md [hunk 2]
```diff
@@ -4,158 +4,238 @@ title: Sourcing from GraphCMS
...
+### Prerequisites
+
+This guide assumes the following:
 
-### Install the boilerplate
+- You have an active GraphCMS account
+- You've created a new GraphCMS project (preferably using the `Blog Starter` template)
 
-To begin, let's create a Gatsby starter site.
+All schema and data references in this guide are from the `Blog Starter` template.
```
Comment (vladar) [inline docs/docs/sourcing-from-graphcms.md:20]:
```suggestion
All schema and data references in this guide are from the GraphCMS `Blog Starter` template.
```

The reason is that Gatsby also has a blog starter and it is a bit confusing here.

File: docs/docs/sourcing-from-graphcms.md [hunk 3]
```diff
@@ -4,158 +4,238 @@ title: Sourcing from GraphCMS
...
-Additionally, you need the `gatsby-source-graphql` library. Because GraphCMS uses GraphQL natively, you will take advantage of Gatsby's ability to simply stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
+In order to fetch data from your GraphCMS project, you will need to install [`gatsby-source-graphcms`](https://github.com/GraphCMS/gatsby-source-graphcms).
 
-You can install this component with:
+You can install this package with:
 
 ```shell
-  # Optionally with `npm install`
-  npm install gatsby-source-graphql
+  yarn add gatsby-source-graphcms@next
```
Comment (vladar) [inline docs/docs/sourcing-from-graphcms.md:39]: I think we need some note about why `@next` here.

**Review by vladar — COMMENTED** [review]
Nice! 👍 Added a couple of suggestions inline.

**Comment by ynnoj** [comment]
@vladar Added a `beta` note. There'll be a subsequent PR to remove the `@next` release once it's 'stable'.

**Review by vladar — APPROVED** [review]
Thanks!

File: docs/docs/sourcing-from-graphcms.md (hunk 3 above)
Comment (muescha) [reply docs/docs/sourcing-from-graphcms.md:41]: was not the rule to use in docs the `npm install` and not `yarn`?

**Pull request merged**